contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,58 @@
+Enforcement team
+================
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, the enforcement team
+pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. Our main priority is to ensure the safety of our community members.
+The second goal is to help educate the community as a whole to be aware of the CoC
+and how to help implement its spirit throughout the community. In case these goals
+conflict, we will prioritize safety of community members over all other goals.
+
+If you think there is or has been a violation to the code of conduct please contact
+enforcement team or if you prefer contact only individual members of the enforcement team.
+
+Members
+-------
+
+Here are all the members of the Code of Conduct enforcement team. You can contact
+any of them by sending an email or tweet. You could also contact all of them at once
+by emailing **coc@symfony.com**.
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, susciit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+Lukas Kahwe Smith
+`````````````````
+
+Lukas is part of symfony core team and working for Liip. Lorem ipsum dolor sit amet,
+consectetur adipiscing elit. Proin enim nibh, suscipit sed rhoncus quis, blandit
+quis sapien. Ut congue pharetra rhoncus.
+
+| **Email:** smith@pooteeweet.org
+| **Twitter:** @lsmith
+
+About the Enforcement Team
+--------------------------
+
+The :doc:`Symfony project leader </contributing/code/core_team>` appoints enforcement
+team with candidates they see fit. The enforcement team will consist of at least
+3 persons. The team should be representing as many demographics as possible,
```

Comment:
Is it OK to use `3 persons` instead of `3 people` ?

Reply:
Yes ... but I'm wondering if `3 people` is more correct. That's what I'd do ... but I'm not native.

Reply:
I've changed to "3 people". I'll let a native speaker ask for a change if needed.